src/neo/models/movies.js

```diff
@@ -70,13 +70,21 @@ const find = (identifier) => {
     .then(r => manyMovies(r)[0]);
 }
 
-const findAll = (where, page, perPage, orderby, dir) => {
+const _makeWhereString = (where) => {
+  let whereString = `WHERE`
+  whereString = where.genre !== null ? `${whereString} g.genre = '${where.genre}' AND` : whereString
+  whereString = `${whereString} movie.year >= ${where.year.from} AND movie.year <= ${where.year.to}`
+  let relationString = where.genre !== null ? `:FALLS_IN` : ''
```

Comment:
I think I just changed this null to undefined to make sure it does not add this if nothing given as parameter